src/Components/Settings/Settings.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import style from './Settings.module.scss'
+import { SettingsContext } from '../../Context/SettingsContext'
+
+const Settings = () => {
+    const [ settings, setSettings ] = useContext(SettingsContext)
+    
+    const onChange = (event) => {
+        setSettings({ 
+            ...settings, 
+            [event.target.id]: +event.target.value
+        })
```

Comment:
Now that I think about it, I wonder if this plus sign just creates type coercion?